Mathlib/Analysis/Complex/Basic.lean

```diff
@@ -444,56 +444,56 @@ theorem normSq_to_complex {x : ℂ} : norm_sqC x = Complex.normSq x :=
 
 section tsum
 
-variable {α : Type*} (𝕜 : Type*) [RCLike 𝕜]
+variable {α : Type*} (𝕜 : Type*) [RCLike 𝕜] {L : SummationFilter α}
 
 @[simp]
-theorem hasSum_conj {f : α → 𝕜} {x : 𝕜} : HasSum (fun x => conj (f x)) x ↔ HasSum f (conj x) :=
+theorem hasSum_conj {f : α → 𝕜} {x : 𝕜} : HasSum (fun x => conj (f x)) x L ↔ HasSum f (conj x) L :=
   conjCLE.hasSum
 
-theorem hasSum_conj' {f : α → 𝕜} {x : 𝕜} : HasSum (fun x => conj (f x)) (conj x) ↔ HasSum f x :=
+theorem hasSum_conj' {f : α → 𝕜} {x : 𝕜} : HasSum (fun x => conj (f x)) (conj x) L ↔ HasSum f x L :=
   conjCLE.hasSum'
 
 @[simp]
-theorem summable_conj {f : α → 𝕜} : (Summable fun x => conj (f x)) ↔ Summable f :=
+theorem summable_conj {f : α → 𝕜} : Summable (fun x => conj (f x)) L ↔ Summable f L :=
   summable_star_iff
 
 variable {𝕜} in
-theorem conj_tsum (f : α → 𝕜) : conj (∑' a, f a) = ∑' a, conj (f a) :=
+theorem conj_tsum [L.NeBot] (f : α → 𝕜) : conj (∑'[L] a, f a) = ∑'[L] a, conj (f a) :=
```

Comment:
Is it possible to drop L.NeBot in this proof? Sketch: if L is bot, then either f has finite support and the tsum is the sum, so the result is true. Otherwise, the tsum is 0 (by the choice to use zero whenever possible), so the result is obviously true.

It would probably good to have a lemma about tsums over L=bot, saying that it's the finsum is the support of f is finite, and 0 otherwise, to streamline this kind of argument. I think this would make dropping L.NeBot possible in many places.

Reply:
I have removed the NeBot assumption in this lemma. With the definitions we have now, tsum for the bot filter is actually _always_ equal to the finsum.